derivation.nix

```diff
@@ -36,6 +36,7 @@ stdenv.mkDerivation {
 
   preCheck = ''
     patchShebangs ./test/
+    cp ${mttn}/traces/*.trace.txt ./test/
```

Comment:
Why `cp` here instead of `install` as in `mttn/derivation.nix`? Also, the behavior I see seems more like `mv` than `cp`.